awx/ui_next/src/screens/InstanceGroup/Instances/InstanceListItem.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import 'styled-components/macro';
+import {
+  Badge as PFBadge,
+  Progress,
+  ProgressMeasureLocation,
+  ProgressSize,
+  DataListAction as _DataListAction,
+  DataListCheck,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+} from '@patternfly/react-core';
+
+import _DataListCell from '../../../components/DataListCell';
+import InstanceToggle from '../../../components/InstanceToggle';
+import { Instance } from '../../../types';
+
+const DataListCell = styled(_DataListCell)`
+  white-space: nowrap;
+`;
+
+const Badge = styled(PFBadge)`
+  margin-left: 8px;
+`;
+
+const ListGroup = styled.span`
+  margin-left: 12px;
+
+  &:first-of-type {
+    margin-left: 0;
+  }
+`;
+
+const DataListAction = styled(_DataListAction)`
+  align-items: center;
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: 40px;
+`;
+
+function InstanceListItem({
+  instance,
+  isSelected,
+  onSelect,
+  fetchInstances,
+  i18n,
+}) {
+  const labelId = `check-action-${instance.id}`;
+
+  function usedCapacity(item) {
+    if (item.enabled) {
+      return (
+        <Progress
+          value={100 - item.percent_capacity_remaining}
+          measureLocation={ProgressMeasureLocation.top}
+          size={ProgressSize.sm}
+          title={i18n._(t`Used capacity`)}
+        />
+      );
+    }
+    return <span css="color: red">{i18n._(t`Unavailable`)}</span>;
```

Comment:
this should use a patternfly CSS variable instead of `red` (`color: var(--pf-global--danger-color--200);`) https://pf4.patternfly.org/documentation/overview/global-css-variables